plugins/hybrid_analysis/icon_hybrid_analysis/actions/lookup_hash/action.py

```diff
@@ -14,30 +16,24 @@ def __init__(self):
             output=LookupHashOutput(),
         )
 
-    def normalize(self, result):
-        formatted = {"found": False, "threatscore": 0, "reports": []}
-        if result.get("response_code"):
-            return formatted
-
-        response = result.get("response", [])
-        if result and len(response) > 0:
-            formatted["found"] = True
-            formatted["reports"] = response
-            if response and isinstance(response, list) and response[0] and isinstance(response[0], dict):
-                formatted["threatscore"] = response[0].get("threatscore")
-            else:
-                formatted["threatscore"] = 0
-
+    def __normalize(self, result):
+        formatted = {Output.FOUND: False, Output.THREATSCORE: 0, Output.REPORTS: []}
+        if result and len(result) > 0:
```

Comment:
I think the `len(result)> 0` condition is unnecessary in this IF statement because the `if result` will check if the array is empty. You can also add a condition to check if `result` is an array.

```suggestion
        if result and isinstance(result, list):
```